teuthology/lock/cli.py

```diff
@@ -149,7 +150,11 @@ def main(ctx):
                 ctx.machine_type, ctx.os_type, ctx.os_version):
             log.error('Invalid os-type or version detected -- lock failed')
             return 1
-        reimage_types = teuthology.provision.fog.get_types()
+        reimage_types = None
+        if config.get('provision_system', 'FOG') == 'pelagos':
+            reimage_types = teuthology.provision.pelagos.get_types()
+        else:
+            reimage_types = teuthology.provision.fog.get_types()
```

Comment:
I think this code is not related to this PR

Reply:
removed